subsys/net/ip/Kconfig.debug

```diff
@@ -26,7 +26,7 @@ module-dep = NET_LOG
 module-def = LOG_LEVEL_DBG
 module-str = Max network stack logging level
 module-help = Max log level. This overrides individual networking log levels.
-source "subsys/net/Kconfig.template.log_config.default.net"
+source "subsys/net/Kconfig.template.log_config.net"
```

Comment:
Funny that in another recent commit, 905106863001f2f4cd202279cc0a8a11022ef14b, there was the opposite change (to another Kconfig file, yeah).